sesman/chansrv/chansrv_fuse.c

```diff
@@ -1084,20 +1101,22 @@ static void xfuse_dump_fs()
  * @param xino xinode structure to dump
  *****************************************************************************/
 
+#if 0
 static void xfuse_dump_xrdp_inode(struct xrdp_inode *xino)
 {
     log_debug("--- dumping struct xinode ---");
     log_debug("name:          %s", xino->name);
-    log_debug("parent_inode:  %ld", xino->parent_inode);
-    log_debug("inode:         %ld", xino->inode);
+    log_debug("parent_inode:  %ld", (long)xino->parent_inode);
```

Comment:
Why? Did you get a warning? What's your compiler? Unneeded casts are evil, then can hide issues.